app-experimental.tex

```diff
@@ -3198,3 +3198,371 @@ \section{Tagged Physical Memory Attribute}
+\end{itemize}
+
+\subsection{Approach}
+
+\begin{itemize}
+\item Definitions:
+\begin{itemize}
+\item Protection level: Some code that sets up its own level scheme. A protection level can have more and less privileged protection levels relative to it. A protection level can be a privilege level (in the common ISA understanding, e.g., supervisor mode), but it can also be more fine-grained.
+\item Current capability: A capability with CL and SL fields that follow the capability level semantic defined by the protection level in execution.
+\item Foreign capability: A capability with CL and SL fields that may follow an arbitrary capability level semantic, defined by a protection level from which the current protection level borrowed the capability.
+\end{itemize}
+\item Architectural state:
+\begin{itemize}
+\item 1 foreign authorisation (FA) bit: This bit determines whether a capability can set or clear (also known as “toggle”) the F bit. The FA bit is a hardware permission bit, which means that it follows monotonicity rules. The FA bit can be cleared, but it can never be set.
+\item 1 foreign (F) bit: This bit determines whether the capability is foreign. Foreign capabilities are “super global” and can be stored by every non-foreign -- also known as “current” -- capability. The setting and clearing of this bit is a privileged operation.
+\item X bits for the capability level (CL) field. The capability level of a capability determines through which other capabilities it can be stored.
+\begin{itemize}
+\item The bits are ordered where “-1” is the most global value and “0” is the most local value
+\item The value of X is determined by the instantiation of the ISA
+\end{itemize}
+\item X bits for the store level (SL) field. The store level determines which other capabilities this capability can store
+\begin{itemize}
+\item The bits are ordered where “-1” is the most permissive value and “0” is the least permissive value
+\item The value of X must be the same as for the capability level
+\end{itemize}
+\item 1 elevate level (EL) bit. The elevation level permission bit determines whether the capability level a capability loaded is reduced to that of the authorising capability.
+\item If this extension is not implemented, the bits taken by this extension have to be all “0”s for the FA, F, SL, and EL fields. For the CL bit the value is “-1”. This leads to software producing global capabilities, but the capabilities cannot be used to store other capabilities. Therefore, software that is oblivious to capability levels runs without any limitation to the flow of capabilities and does not pose any security risks.
```

Comment:
I find this paragraph unclear. 

> This leads to software producing global capabilities, but the capabilities cannot be used to store other capabilities.

Surely we don't want this to be the case? I think when F=0 CL="-1" SL=0 means that the global capabilities produced can store any other global capabilities.